src/db/units.ts

```diff
@@ -175,4 +180,14 @@ export async function nearbyUnits(position: PositionLL, accuracy: number) {
     ])
     .whereNull('destroyedAt')
     .whereNull('goneAt')
+
+  // if not Coalition_ALL, search by country
+  if (Coalition.COALITION_ALL !== coalition) {
+    const country = countryFrom(coalition)
+    query = query.where({ country })
+  }
+
+  const nearby = await query
+
+  return nearby
```

Comment:
```suggestion
  return  await query

```

Reply:
`await` is what triggers the knex builder pattern to close and initiate the request, but we don't need an extra local var